recipes/bioconductor-bsgenome/meta.yaml

```diff
@@ -1,42 +1,58 @@
 package:
   name: bioconductor-bsgenome
-  version: 1.40.1
+  version: "1.42.0"
+
 source:
-  fn: BSgenome_1.40.1.tar.gz
-  url: http://bioconductor.org/packages/3.3/bioc/src/contrib/BSgenome_1.40.1.tar.gz
-  md5: 52fcebf91ba05468fa6c8320f45a0876
+  fn: BSgenome_1.42.0.tar.gz
+  url: https://bioconductor.org/packages/release/bioc/src/contrib/BSgenome_1.42.0.tar.gz
+  md5: ee6db342fe767fc8092ce88c0d5d15cc
+
 build:
   number: 0
   rpaths:
     - lib/R/lib/
     - lib/
+
+#Suggests: BiocInstaller, Biobase, BSgenome.Celegans.UCSC.ce2,
+# BSgenome.Hsapiens.UCSC.hg38, BSgenome.Hsapiens.UCSC.hg38.masked,
+# BSgenome.Mmusculus.UCSC.mm10, BSgenome.Rnorvegicus.UCSC.rn5,
+# TxDb.Hsapiens.UCSC.hg38.knownGene, TxDb.Mmusculus.UCSC.mm10.knownGene,
+# SNPlocs.Hsapiens.dbSNP141.GRCh38, XtraSNPlocs.Hsapiens.dbSNP141.GRCh38,
+# hgu95av2probe, RUnit
+
+#I specify the versions of respective dependencies while buiding the package,
+#whereas in the comments, I document
+#the version ranges specified in the origianl R description file.
```

Comment:
That would be good! Also, see #2816, genomation is in there.

 `rhtslib` needs work because autoconf is not working. Will likely need the path fixes in https://github.com/bioconda/bioconda-recipes/blob/fd57b3a52bf94a41e783424f1db3364f66be7bbe/recipes/sailfish/build.sh